deploy/kustomize/patches/clusterserviceversion-patch.yaml

```diff
@@ -0,0 +1,169 @@
Q/TN0GbHBKR7m9+jMjPixmfBZj8SKJOjcuoO0HiUeVTapDYO/LT+W4rCIzKDy9nbJiVhKGsbKozsqp/6NNEET7dOJrXaRrdyhI9VlZhNr0zFSyuNbf0mfVRDOveWYXD2d9G246JMK9SUIS/FkqAa8fPJdvxq8iqU1nrx0Ix3R0hS1tJ3pmc/frQrZ45q1MeP5oXx4eeojMcEsnNUqZxHknPmqIYzh1W5gTzcksDzb+/CJ26bbIAwHh+/ZRJu/sti3DtlNb7+4Dwt7mAw1/x6FhZtKcWlv5yebWsUcJf+8iVYRs4VyX4WLF9YvICE4ak8fvTDgqH2UWUPecmcZjImDFUg+QM/CYQLT/dNfUeUaI1B3vk/n4KiRTvx14Xb8fxb2/HewVpc+8hryoaPF+uzd79E6m28lCjNLS7CV8nOLNlaCsvIx4+qc0U6NRi5RlmcyceP6mHo51GpI5AKPUBjNsoiM49q5CnY0dxb1W6wmrKSJ/XNh18TaZCzTKLuG//4pmi4/tIDr+Jccn0fm72BoNRIGOOkJJx9c0aVqevbj87HrmNW+fwQ/TPE9UNejO7tmXj86KlIRiHDwfJNajObnlDQmY1hpdjEs7z6+rG51IFL9KpmnC4Von5OH59buA1vbjwqvKsvPTBbdFNaRtQ95GuwLbnpyYWooIBYnqeMwSLd2x5FOroEEGHEC3jiZSGPH7XrniZgfPyo2bjV0WCc2iiLnpxqKnfCUJauO3vSeG1DKT5378ua6lHtBXtWnMGVqXQJ5FNkwGes3IfJxbvFx7snrjQx3kW48BdT8cgra+Gpb1A8RAVIBob6QPfOnENexvrES6N0FPL4UePYPOOk7dGGvOhhZNRUKmfVL2cjNM9m6egWDQsX3Tkt466OU430eNxFm128vRzjyIhzh6Ja42A3eMLinbjtb8WmHtUVv3oJL63Yg0C4SQGhLgKiJgxZMrQnl7XL00zqBpwpGKczHEwPQz+PSp0SNHJSmzo9QZ5o0sPgZe5RKQ+aISD+xrjY6HOEvch4SpZxGc/obnJlF24qxeKt5Zj9zgFcLMAViRzWeeSBnT1ugiFQfAEX/8c0rNpVLhKhKgT9sqh2Y2yT2swfsvj/i3srpcO8BJv3eBm7tRTslTsjuOGJN5Wad5HOS+LPJ2jG/LxbJ2HfCRd+8cJybD5ch0NkL254fIEOWgYGOwJf/d0cHKl2yevgpkEDDA2I6lHlgpHrSHK+J16eCgzjPKpcY/PyTfjUl2DHAoOlom9gEOsP2fCNh18bkTL/CN3tNzzxhmgP1ewIrZkUgS/dcRxX3fcKrn/8DREEGqNxjlfumrAcVRSpZ6soEwnJPBIoYXgwr3mhKR8M4zPEzUZZ5HrI4mgPy9LPMMw95EUdf9STNT2hkLN+ia4Upq08gMsoTshOkReJTsR7Jq/G1GV7Rc+uPti7/g9vYN66wyIgPGtEEapIRPJ/fm0Tat1BA4CRMAQQfQk2H4yQkqPy13+4jx81PrvECCPfIGKz8Ue5YOgzt83tXXh09ka6kw09UspiW7K2pIakYza+/6eFZKyXaHEI91V9meIOtfaR8cSKxCmq19YeQLAharr5pkDiBY3NyyQMcz222ti6M5Y+qlOd1JYr1ii0j4phBJrbRZpDbOg4abSNRaRP3TEVCzaWYs2eSryzr0oUn3ido6uTSwgviI8cibMkbTpYLer4uaTBFIiwG3lKsIUnDP9FRjznXBFzuyEPXfbL0SGq3VDS5geq/bjhjwt1Bnu8ElWPx+W/mklGeK7WzHYlxSGbyHD/9c3tuPfF1Xh2wVaKOaYYclLSeP/oqbew42id6HjhJ4kWCkPaELrA1gJLsP8vm9rYbpgHfl15Zxhq0sEFOGHEh9Dd24fiHRW4Rm1w1tUvONj7wZNvYdqK/XhnfzVue26ptulfemAO1h+oxeNzNop+3EwuarwG7v6pq1Fa68lyRApdfI0Wzu20FOhR5WpqG+sTL/M9ZHFsk9oydkMPQ51HpVdTAgbFGT0EY+aag0qwZygWjXsBn7vnZdGgcNNTbwtJmLn6gK6oVCTiCDXlnkm7F4nAcFLxTgr2GsekovSLSwIWnlGSrwT7YcIwe3ZJIUY8mcydo1KlIwvGwKDSnd4jIm+RbTVtOCgSLurU5fvwCkHjbO06kohfkYfFIDL2IpNCYdvDwwLmvnsAjc2tOrd27ECYheUf//iHKOjk6qM6HRiFTmozez5TzEQyzCZD55pHxXZDjiocFOcxKt0NuKNoRd6+KHWzOaZYs7cKLy7dgx//5W08vWALvv3IfNp8nRc1Ttqbbz40D0u3laGRruNUIKg5LC4xMwvLP//5T1EhYwnJNxwsH4zRHj+qAjFKhvFhWcYnl+UeDtZVwHAwqabYeO8sd4nIe2SZ1SwLK8Fwmn3ToVrlIE1RJoGoqCmWjFv+Wox9xx1544qCoNA1s6PBLCygf0yGg8N8MESDggGGXjpGG5tnpqbGMsOw0EHEmqpSDPg7+2vJRZ0pNvOsm4tG6Rx8IZMmIWlgw32uGvAJ6ZAtPB8hT+rBGWth94bkBKO2ttMCwq1IzID/CSD873/+539EfTrfpLZ/5XCw9pyp9FTu4WAKDG5YSyR7xGgLi9KgNmqT86iSIxenRp6iyLs+0qRVUU9VTamq6u9//0DFkAGiQuGYhLsEvR8CDLOnwRlhmOWo9DCMEz717q3oNqfljrThsVc34dxbJ2le0OkBkevz971Mxr5ETKCQM7ROT02xZPz973/XI8gGwv8++OAfok+JT8eeDgzjU2gKkYzcTxNQx+b15i3BsmRUextEg/O5t0w2yS2d2mJ1d/3jC7C+pFobZiZWa+spwWCpYJuhqqm8QPgfG5cPPvhAnAvh3ifedD0Mp8tzypKhf+ZV/hJsUvOmOBIvZIbh3goPrr5/1og6xCmDUGD86M9v4SgFe6LNSHE8xhz0tSdEnMGuLYPgPTb7938BhWWtdubpXywAAAAASUVORK5CYII=
```

Comment:
Please explain where this binary blob came from. Don't include URLs to internal services, though.

Reply:
Done.